### DISCUSSION TITLE
Add "Group disconnections within 5 minutes" to Log Options.

### DISCUSSION THREAD

**Discussion by omerhochman** [InternetConnectionMonitor/icm-public#93] [Ideas and Feature requests]
I have a lot of disconnections sessions with my new network and I would like to have a nice clean table 

Some like this
Date and time | Status | Status duration
-- | -- | --
04/30/2023 9:13:39 PM | Online | 0:10:05
04/30/2023 9:10:37 PM | No Internet connection. On and Off | 0:03:02
04/30/2023 9:08:51 PM | Online | 0:01:46

**Instead of the something like that**
<img width="640" alt="Screenshot 2023-04-30 at 22 42 22" src="https://user-images.githubusercontent.com/6758175/235373170-666523b4-404e-4fb9-9609-0b8dbcb1ad2b.png">

I suggest adding "Group disconnections by X time" functionality to Log Options.

**Comment by dmishh** [comment]
@omerhochman thanks, that's a cool feature proposal 👍
